internal/service/common/db/migration.go

```diff
@@ -96,6 +96,12 @@ func NewHandler(cfg MigrationConfig) (*MigrationHandler, error) {
 		connStr += fmt.Sprintf("&x-migrations-table=%s", cfg.MigrationsTable)
 	}
 
+	if _, err := os.Stat(utils.DefaultServiceCAFile); err == nil {
```

Comment:
why do we need this?

Reply:
Why do we need the `os.Stat(...)` check?   I had to add this to make sure that standalone testing isn't affected.  Once we get enough code in-place so that everyone can test in-cluster then we won't need this.

Reply:
shouldn't we fail if this file is not found and sslMode is `verify-full`?

Reply:
It will fail on its own because the database connection won't initialize.

Reply:
yeah, I was thinking about a more specific log. Buy, yes it doesn't matter much